src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
```

Comment:
backward compatibility should not be affected - timerChannelConfig_t is converted into existing structure in both cases.

Future change can be done quite easily for AF+inverted (simply dropping them. It will only waste PG space)
DMA representation change is possible, but will need transition code.